specification/cognitiveservices/Language.AnalyzeText/main.tsp

```diff
@@ -0,0 +1,103 @@
+import "./models/custom.text.tsp";
+import "./models/entity.linking.tsp";
+import "./models/entity.recognition.tsp";
+import "./models/healthcare.tsp";
+import "./models/key.phrase.extraction.tsp";
+import "./models/language.detection.tsp";
+import "./models/pii.entity.recognition.tsp";
+import "./models/sentiment.analysis.tsp";
+import "./models/summarization.tsp";
+import "./models/custom.healthcare.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+using Azure.ClientGenerator.Core;
+
+@useAuth(
+  TypeSpec.Http.ApiKeyAuth<
+    TypeSpec.Http.ApiKeyLocation.header,
+    "Ocp-Apim-Subscription-Key"
+  >
+)
+@doc("The language service API is a suite of natural language processing (NLP) skills built with best-in-class Microsoft machine learning algorithms.  The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction, language detection and question answering. Further documentation can be found in <a href=\"https://docs.microsoft.com/azure/cognitive-services/language-service/overview\">https://docs.microsoft.com/azure/cognitive-services/language-service/overview</a>.0")
+@service({
+  title: "Microsoft Cognitive Language Service - Text Analysis",
+})
+@server(
+  "{Endpoint}/language",
+  "Single server endpoint",
+  {
+    @doc("Supported Cognitive Services endpoint (e.g., https://<resource-name>.api.cognitiveservices.azure.com).")
+    Endpoint: string,
+  }
+)
+@versioned(Versions)
+namespace Language.AnalyzeText;
+
+@doc("Analyze Conversations Service Versions")
+enum Versions {
+  @doc("Version 2022-05-01")
+  @useDependency(Azure.Core.Versions.v1_0_Preview_2)
+  v2022_05_01: "2022-05-01",
+
+  @doc("Version 2023-04-01")
+  @useDependency(Azure.Core.Versions.v1_0_Preview_2)
+  v2023_04_01: "2023-04-01",
+
+  @doc("Version 2023-11-15-preview")
+  @useDependency(Azure.Core.Versions.v1_0_Preview_2)
+  v2023_11_15_preview: "2023-11-15-preview",
+}
+
+alias ServiceTraits = NoRepeatableRequests &
+  NoConditionalRequests &
+  NoClientRequestId;
+
+alias Operations = Azure.Core.ResourceOperations<ServiceTraits>;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@post
+@route("/:analyze-text")
+@doc("Request text analysis over a collection of documents.")
+op analyzeText(
+  @doc("Collection of documents to analyze and a single task to execute.")
+  @body
+  body: AnalyzeTextTask,
+
+  ...AnalyzeTextQueryParameters,
+): AnalyzeTextTaskResult | ErrorResponse;
+
+#suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+@doc("Get the status of an analysis job. A job can consist of one or more tasks. After all tasks succeed, the job transitions to the succeeded state and results are available for each task.")
+@summary("Get analysis status and results")
+op analyzeTextJobStatus is LanguageLongRunningResourceRead<
+  AnalyzeTextJobState,
+  AnalyzeTextJobQueryParameters
+>;
```

Comment:
Should be: 
```suggestion
op analyzeTextJobStatus is languageOperations.ResourceRead<
  AnalyzeTextJobState,
  AnalyzeTextJobQueryParameters
>;
```

Reply:
Please note you'll need for define the following earlier in your file: 

```
alias languageOperations = ResourceOperations<{}, ErrorResponse>;
```

Reply:
Please make sure to add any headers you do support such as client request id or repeatable requests in the ResourceOperations definition.